go/vt/vtgate/planbuilder/operator_transformers.go

```diff
@@ -734,3 +777,22 @@ func transformLimit(ctx *plancontext.PlanningContext, op *operators.Limit) (logi
 
 	return createLimit(plan, op.AST)
 }
+
+func createLimit(input logicalPlan, limit *sqlparser.Limit) (logicalPlan, error) {
+	plan := newLimit(input)
+	pv, err := evalengine.Translate(limit.Rowcount, nil)
+	if err != nil {
+		return nil, vterrors.Wrap(err, "unexpected expression in LIMIT")
+	}
+	plan.elimit.Count = pv
+
+	if limit.Offset != nil {
+		pv, err = evalengine.Translate(limit.Offset, nil)
+		if err != nil {
+			return nil, vterrors.Wrap(err, "unexpected expression in OFFSET")
+		}
+		plan.elimit.Offset = pv
+	}
+
+	return plan, nil
```

Comment:
Shouldn't we use `vterrors` errors here?

Reply:
I just moved this method from `postprocess.go`. It's not wrong to use `Wrap`, right?